src/Analysis/GWASV2/GWASContainer.jsx

```diff
@@ -15,41 +15,40 @@ const GWASContainer = () => {
     setSelectedStudyPopulationCohort,
   ] = useState({});
   const [selectedControlCohort] = useState(undefined);
-  const [selectedCaseCohort] = useState(undefined);
-  const [selectedCovariates] = useState([]);
-  const [selectedDichotomousCovariates] = useState([]);
+  const [newCovariateSubset] = useState([]);
+  const [outcome] = useState({});
```

Comment:
I think I prefer what is used 2 lines above:
```
useState(undefined)
```
It makes the code more readable and avoids things like:
```
Object.keys(selectedStudyPopulationCohort).length === 0
```